README.md

```diff
@@ -1,67 +1,58 @@
 
-Verified Hardware Platforms:
+#### Intel Data Center GPU
 
-* Intel® Data Center GPU Max Series, Driver Version: [803](https://dgpu-docs.intel.com/releases/LTS_803.29_20240131.html)
+* Intel® Data Center GPU Max Series
 
-#### Software Requirements
+#### Intel Client GPU
 
-* Ubuntu 22.04, SUSE Linux Enterprise Server(SLES) 15 SP4
-  * Intel® Data Center GPU Max Series
-* Intel® oneAPI Base Toolkit 2024.0
+* Hardware Verified with Windows® 10 and 11
+  * Intel® Core™ Ultra Processors with Intel Arc Graphics
+  * Intel Core Ultra Processors Series 2 with Intel Arc Graphics
+  * Intel Arc B-Series Graphics
+* Hardware Verified with Ubuntu 24.10
+  * Intel Arc A-Series Graphics
+  * Intel Core Ultra Processors with Intel Arc Graphics
+  * Intel Core Ultra Processor Series 2 with Intel Arc Graphics
+* Hardware Verified with Ubuntu 24.04 and Windows* Subsystem for Linux 2 (WSL2) with Ubuntu 24.04
+  * Intel Arc A-Series Graphics
+  * Intel Core Ultra Processors with Intel Arc Graphics
 
-#### Install Intel GPU Drivers
+### Software Requirements
 
-|OS|Intel GPU|Install Intel GPU Driver|
-|-|-|-|
-|Ubuntu 22.04, SLES 15 SP4|Intel® Data Center GPU Max Series|  Refer to the [Installation Guides](https://dgpu-docs.intel.com/installation-guides/index.html#intel-data-center-gpu-max-series) for latest driver installation. If install the verified Intel® Data Center GPU Max Series/Intel® Data Center GPU Flex Series [803](https://dgpu-docs.intel.com/releases/LTS_803.29_20240131.html), please append the specific version after components, such as `sudo apt-get install intel-opencl-icd==23.43.27642.38-803~22.04`|
+For the hardware and software prerequiste, please refer to [PyTorch Prerequisites for Intel GPUs](https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpus.html) for Intel GPU Driver and Intel Deep Learning Essentials.
 
-#### Install oneAPI Base Toolkit Packages
+* Intel GPU Driver: Install Intel GPU drivers along with compute and media runtimes and development packages.
+* Intel Deep Learning Essentials: Install a subset of Intel® oneAPI components needed for building and running PyTorch.
 
-Need to install components of Intel® oneAPI Base Toolkit:
 
-* Intel® oneAPI DPC++ Compiler
-* Intel® oneAPI Math Kernel Library (oneMKL)
-* Intel® oneAPI Threading Building Blocks (TBB), dependency of DPC++ Compiler.
+## Build
 
-```bash
-wget https://registrationcenter-download.intel.com/akdlm//IRC_NAS/20f4e6a1-6b0b-4752-b8c1-e5eacba10e01/l_BaseKit_p_2024.0.0.49564.sh
-# 2 components are necessary: DPC++/C++ Compiler and oneMKL
-sudo sh l_BaseKit_p_2024.0.0.49564.sh
-
-# Source OneAPI environment
-source /opt/intel/oneapi/compiler/2024.0/env/vars.sh
-```
-
-
-## 3. Build
-
-Need to built this project as a submodule of PyTorch, after install Intel GPU Driver and oneAPI Base Toolkit Packages.
+Need to built this project as a submodule of PyTorch, after install Intel GPU Driver and Intel Deep Learning Essentials.
```

Comment:
Intel Deep Learning Essentials, double check, is this the formal name? or miss @